components/locale-provider/__tests__/index.test.tsx

```diff
@@ -60,6 +60,7 @@ import 'dayjs/locale/tk';
 import 'dayjs/locale/tr';
 import 'dayjs/locale/uk';
 import 'dayjs/locale/ur';
+import 'dayjs/locale/uz-latn';
```

Comment:
```suggestion
import 'dayjs/locale/uz';
```

<img width="635" alt="image" src="https://user-images.githubusercontent.com/6134547/207536397-b203c699-7041-45c0-b97f-17f4d742b60f.png">

Reply:
In Uzbekistan is used two types of letters Cyrillic (uz) & Latin (uz-latn) for the Uzbek language.
Nowadays, a major number of websites and applications are using the **Latin version** of the Uzbek language. 

I agree with your proposed changes IF we switch the language in dayjs:
- `uz` rename to `uz-cyr`
- `uz-latn` rename to `uz`

Reply:
The implementation of `dayjs` locale is consistent with the style of `moment` locale. If you want to add `uz-latn` but `uz`, use `uz-latn` directly.

`https://github.com/moment/moment/blob/develop/locale/uz-latn.js`
`https://github.com/moment/moment/blob/develop/locale/uz.js`

Reply:
Add we need to reconsider whether the PR you sent to v4 https://github.com/ant-design/ant-design/pull/39353 belongs to `uz` or `uz-latn`

Reply:
> `uz` rename to `uz-cyr`

There are only two options for now, `uz` or `uz-latn`, no `uz-cyr`.

Reply:
Yep, I got your point. But what was done at `momentjs` is currently outdated:

1)  Schools and universities teach through Uzbek books in the Latin alphabet.

2) If we take browser into or any popular Intl lib e.g. formatjs then they use Latin characters for `uz`:
https://unpkg.com/@formatjs/intl-displaynames@6.2.3/locale-data/uz.js

<img width="586" alt="Screenshot 2022-12-15 at 11 01 36" src="https://user-images.githubusercontent.com/1946920/207831538-83430788-ee2a-476b-8f20-a8304c3075bb.png">
They are using `uz-latn` & `uz-cyrl` for exact situations, and `uz` in Latin for default widespread use cases.

3) Currently antd repo does not have any case of multiple alphabet support for translations. How should localization be named `uz_UZ-latn` or  `uz_UZ_latn`?

Moreover, I need to update `rc-pagination` and `rc-picker` packages for introducing alphabets.

---

I still insist on 
- keeping the Uzbek language in the Latin alphabet for uz_UZ locale in `antd` & `rc-componets` repos.
- open PR to dayjs to rename
  - `uz` -> to `uz-cyrl`
  - keep `uz-latn` untouched
  - in `uz` file put latin alphabet translations from `uz-latn`

`dayjs` must be updated to meet the current situation with the Latin alphabet in Uzbekistan. I'm ready to open PR there.

Reply:
How is `antd` compatible with users who still insist on using `momentjs` when we make this change? Ref: https://ant.design/docs/react/use-custom-date-library

Reply:
@iamkun @zombieJ Let's talk about this.

Reply:
Hmm... I have no idea about this. @iamkun

Reply:
Maybe we should keep the same with moment.js for a while.

And switch to the "may be better" locale in the next major version?

Reply:
Fix in dayjs for `uz` locale, which was added 7 years ago:
https://github.com/iamkun/dayjs/pull/2170

Reply:
You may accept this PR. 
It will be compatible with the current and the next version of dayjs.